core/kg-query/src/main/java/it/unibz/inf/ontop/query/translation/impl/RDF4JTupleExprTranslator.java

```diff
@@ -514,7 +624,7 @@ private TranslationResult translateJoinLikeNode(BinaryTupleOperator join) throws
         if (join instanceof LeftJoin) {
             Sets.SetView<Variable> variables = Sets.union(leftTranslation.iqTree.getVariables(), rightTranslation.iqTree.getVariables());
             Optional<ImmutableExpression> filterExpression = Optional.ofNullable(((LeftJoin) join).getCondition())
-                    .map(c -> topSubstitution.apply(getFilterExpression(c, variables)));
```

Comment:
Why ignoring the EXISTS map?

Reply:
At the least we should throw an unchecked exception if we are in a case we don't support